notebooks/openvino_inference.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<p>It might be possible to quiet this first warning by updating the listed requirements, but it's not a big deal if you don't.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/214/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>